UI/AdvancedXBind/AdvancedXBind/AdvancedXBind/AdvancedXBind/Platforms/WebAssembly/LinkerConfig.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="AdvancedXBind" />
```

Comment:
Original contents should be restored. Except for `AdvancedXBind.Wasm` it will be `AdvancedXBind` as it's already.

Reply:
@morning4coffe-dev can you check @Youssef1313 comment please

Reply:
Done. I will keep this sample as draft, I need to test on other platforms as well.